frontend-react/src/components/header/HIWDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import { Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { useEffect, useState } from "react";
+
+const HIWDropdown = () => {
```

Comment:
nit: preference for descriptive components like `HowItWorksDropdown` since HIW might not be a clear acronym